File: submissions/sm_021_piyush-jain/week_4/day_1/session1/checkSuperset.js
var arr1=[1,5,3,4,8]
var arr=[1,5,3,4]
function checkSupersetarray(arr,arr1)
{   var count=0
    var obj={}
    for(var i=0;i<arr1.length;i++)
    {
        obj[arr1[i]]=1;
    }
    console.log(obj)
    for(var j=0;j<arr.length;j++)
    {
        for(key in obj)
        {   
            if(Number(key)==arr[j])
            {   
                count+=1;
            }
        }
    }
    if(count==arr.length)
        console.log("Array1 is superset of array2")
    else 
        console.log("array1 is not superset of array2")    
}
checkSupersetarray(arr,arr1)

File: submissions/sm_104_asheeh/week_13/day_4/session1/check_superset.py
# check if the set1 is the superset of the set2
set1 = {2,5,6,78,9} 
set2 = {2,5,6}
count = 0
for i in set2:
    for j in set1:
        if i == j:
            count += 1
            break;
if count == len(set2) and len(set1) > len(set2):
    print("set1 is the superset of set2.")  
else:
    print("set1 is not the superset of set2")